src/helpers/getCountriesFromRegions.ts

```diff
@@ -6,7 +6,7 @@ interface IProps {
   countries: TGeography[];
 }
 
-export const getCountriesFromRegions = ({ regions, countries, selectedRegions }: IProps) => {
+export const getCountriesFromRegions = async ({ regions, countries, selectedRegions }: IProps) => {
```

Comment:
Question: Does this function need to be asynchronous? I don't see any promises or any `await` within it